app/src/main/java/com/github/pierry/arctouchcallenge/common/DateHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.github.pierry.arctouchcallenge.common;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class DateHelper {
+
+  private static Date parseDate(String date) {
+    ArrayList<String> formats = new ArrayList<String>();
+    formats.add("yyyy-MM-dd'T'HH:mm:ss'+0000'");
+    Date extractedDate;
+    for (String format : formats) {
+      SimpleDateFormat dateFormat = new SimpleDateFormat(format);
+      try {
+        extractedDate = dateFormat.parse(date);
+        return extractedDate;
+      } catch (ParseException ignored) {
+        System.out.println(ignored.getMessage());
```

Comment:
Yes, sorry for this. This class is old, and I use always when I need work with dates.
I know native log - `Log.x();` and other libraries, like Timber.